helm/charts/nats-kafka/Chart.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v2
+version: "0.1.0"
```

Here I put the version of the charts in the repo so would be 0.8.4
```suggestion
version: "0.8.4"
```